client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -124,5 +158,21 @@ export default defineComponent({
         </v-row>
       </v-col>
     </v-container>
+    <v-snackbar
+      v-model="snackbar"
+      :timeout="-1"
+      color="error"
+    >
+      {{ errorText }}
+      <template v-slot:action="{ attrs }">
+        <v-btn
+          text
+          v-bind="attrs"
+          @click="snackbar = false"
+        >
+          Close
+        </v-btn>
+      </template>
+    </v-snackbar>
```

Comment:
kudos for using `v-snackbar` and not further entrenching the snackbar service.

Reply:
This could in the future be moved outside of `Recent.vue` into a more global location for the Desktop version to return error messages.